src/solvers/dg/3d/dg.jl

```diff
@@ -688,6 +705,48 @@ function init_mortar_connectivity!(elements, mortars, mesh::TreeMesh{3})
 end
 
 
+function init_boundary_conditions(n_boundaries_per_direction, mesh::TreeMesh{3})
+  # "eval is evil"
+  # This is a temporary hack until we have switched to a library based approach
+  # with pure Julia code instead of parameter files.
+  bcs = parameter("boundary_conditions",
+                  ["nothing", "nothing", "nothing", "nothing", "nothing", "nothing"])
+  if bcs isa AbstractArray
+    boundary_conditions = eval.(Symbol.(bcs))
+  else
+    # This adds support for using a scalar boundary condition (like 'periodicity = "false"')
+    boundary_conditions = eval.(Symbol.([bcs for _ in 1:n_directions(mesh.tree)]))
+  end
```

Comment:
You could also use
https://github.com/trixi-framework/Trixi.jl/blob/d067c379f22d0000b17be9937b97a341d526280f/src/solvers/solvers.jl#L11-L12
to allow users to pass their own functions defined outside of Trixi.jl. Same in 2D.

Reply:
Yes, I should do that. Does it support the dot Syntax?

Reply:
It should, the dot syntax is handled automatically, you don't have to do anything special to use it.

Reply:
OK, then I'll put this in and once the tests pass, merge this PR

Reply:
Changed in 8c3b471.